python-project-template/.gitignore

```diff
@@ -141,3 +141,6 @@ tmp/
 
 # Mac OS
 .DS_Store
+
+# Pytest monitor
+.pymon
```

Comment:
Does this use pytest-monitor anymore? Or do I just have `.pymon` in my local environment from other changes?

Reply:
You're right @delucchi-cmu, it does not! The supernova incubator project still does though, that's probably why you have it locally. I will remove this ignore.